src/cli.toit

```diff
@@ -254,9 +277,8 @@ abstract class Option:
 
   If $split_commas is true, then $multi must be true too. Values given to this option are then
     split on commas. For example, `--option a,b,c` will result in the list `["a", "b", "c"]`.
-
   */
-  constructor .name --.short_name --.short_help --required --hidden --multi --split_commas:
+  constructor.from_sub .name --.short_name --.short_help --required --hidden --multi --split_commas:
```

Comment:
This should surely be a private constructor.

Reply:
should be public. Renamed to 'from_subclass'
done.